src/test/resources/pages/instructorStudentRecords.html

```diff
@@ -121,16 +120,13 @@ <h4 class="modal-title">Benny Charlés's Profile - More Info</h4>
                                     </div>
                                     <div id="visibility-options0" class="panel panel-default" style="display: none;">
                                         <div class="panel-heading">Visibility Options</div>
-                                        <table class="table text-center" style="color:#000;">
+                                        <table class="table text-center" style="color: #000; background: #fff;">
```

Comment:
Alternatively, I can remove the second style altogether, since it's already ignored in the first place anyway. Is it better?